docs/conf/writing.md

```diff
@@ -218,8 +218,7 @@ webhookUrl: https://smee.io/foo
 clientId: someGithubAppClientId
 clientSecret: someGithubAppClientSecret
 webhookSecret: someWebhookSecret
-privateKey: |
-  -----BEGIN RSA PRIVATE KEY-----
-  SomeRsaPrivateKey
-  -----END RSA PRIVATE KEY-----
+privateKey: someSecretRsaPrivateKey
```

Comment:
I think it might make sense to keep the example key part like we had before. It shows the format on how to use the `yaml` `|` syntax to make sure that the key is valid.

Reply:
Hey 🙂 I kept the example like it was, and added this information about the yaml syntax (that I didn't know, thank you!)

Is it better this way ?

Reply:
Nice thanks!